### PR TITLE
Add a type and provider to manage sslservices

### PR DESCRIPTION
**Pull request by jovandeginste**
Feedback please: should I use eg. `cipher_details` rather than `cipherdetails`?

### CODE REVIEW

**Comment by tphoney** [comment]
The changes look great. Have you been able to run the acceptance tests, as it would be great to have an acceptance test for this new type/provider ?

**Comment by jovandeginste** [comment]
I'll need to have a look at writing acceptance tests. I'm testing them on our NetScaler directly :-)

File: lib/puppet/provider/netscaler_sslservice/rest.rb [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	def self.instances
+		instances = []
+		sslservices = Puppet::Provider::Netscaler.call('/config/sslservice')
+		return [] if sslservices.nil?
+
+		sslservices.each do |sslservice|
+			instances << new({
+				:ensure              => :present,
+				:name                => sslservice['servicename'],
+				:cipherdetails       => sslservice['cipherdetails'],
```
Comment (tphoney) [inline lib/puppet/provider/netscaler_sslservice/rest.rb]: cipher_details is more apt following what is in https://github.com/puppetlabs/puppetlabs-netscaler/blob/master/lib/puppet/provider/netscaler_csaction/rest.rb#L18-L19

**Comment by tphoney** [comment]
could you squash the commits, the acceptance test can be added later

**Comment by jovandeginste** [comment]
Ok, serious update (including squash); I put the documentation in a separate directory - easier to update and merge in the future ...

**Comment by tphoney** [comment]
@jovandeginste apologies for the delay in looking at this. Thanks for the squash. The documentation should be in the README. I know it is kind of full, but it is standard practice across all puppet / community modules

**Comment by jovandeginste** [comment]
Ok done

**Comment by tphoney** [comment]
Apologies for the late / no response, I have been caught up in an office move and some family things. Unfortunately i will be on vacation for the next couple of weeks.  
I will get back to this as soon as possible.

**Comment by david22swan** [comment]
@jovandeginste Is there any movement on this PR?

**Comment by david22swan** [comment]
@jovandeginste I apologize but due to the amount of tie that has gone by since this PR was last updated and your lack os response to attempts to communicate we feel that we must now close this PR. If at a later time you decide to return to it then we will be more than happy to once again review it.
Best Wishes

**Comment by jovandeginste** [comment]
@david22swan I understand, I will revisit these things when the NetScaler configuration through Puppet is again on my radar. A lot changed in the past years :-)

**Pull request closed without merging**